example.groovy

```diff
@@ -1,3 +1,4 @@
-void sayHello() {
+void sayH ello() {
```

Comment:
📝 **[npm-groovy-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Violation in class None. Parentheses in the 'ello' method call are unnecessary and can be removed.  UnnecessaryParenthesesForMethodCallWithClosure